profile/profile.go

```diff
@@ -115,6 +116,16 @@ func initEventMap(profile *profctx.Profile) {
 			common.REG_ACCEPT_EVENT:      common.REG_COMPLETE_EVENT,
 			common.PROFILE_PASS_EVENT:    common.QUIT_EVENT,
 		}
+	case GUTI_REGISTER:
+		profile.Events = map[common.EventType]common.EventType{
+			common.REG_REQUEST_EVENT:      common.AUTH_REQUEST_EVENT,
+			common.AUTH_REQUEST_EVENT:     common.AUTH_RESPONSE_EVENT,
+			common.SEC_MOD_COMMAND_EVENT:  common.SEC_MOD_COMPLETE_EVENT,
+			common.REG_ACCEPT_EVENT:       common.REG_COMPLETE_EVENT,
+			common.GUTI_REG_REQUEST_EVENT: common.REG_ACCEPT_EVENT,
```

Comment:
Does not sound right, though it was useful for you to do the test ! We should take real scenario how GUTI is used. You may like to de-register and then register again with GUTI.  We need to do some changes in gNBsim architecture to handle these kind of scenarios.

Reply:
yes, i changed a profile and also changed some logic to send guti registration request after deregistration